tests/server/test_params.py

```diff
@@ -89,43 +90,43 @@ def test_parse_geo_arg(self):
             with app.test_request_context("/"):
                 self.assertEqual(parse_geo_arg(), [])
         with self.subTest("single"):
-            with app.test_request_context("/?geo=state:*"):
-                self.assertEqual(parse_geo_arg(), [GeoSet("state", True)])
-            with app.test_request_context("/?geo=state:AK"):
-                self.assertEqual(parse_geo_arg(), [GeoSet("state", ["ak"])])
+            with app.test_request_context("/?geo=fips:*"):
+                self.assertEqual(parse_geo_arg(), [GeoSet("fips", True)])
+            with app.test_request_context("/?geo=fips:{}".format(FIPS[0])):
```

Comment:
nit: shorter, potentially easier to read/understand
```suggestion
            with app.test_request_context(f"/?geo=fips:{FIPS[0]}"):
```
theres a number of these type of potential changes below...  im not going to mark them, but if you decide to change this one, you should probably change those too.